plugins/woocommerce/e2e/tests/admin-tasks/payment.spec.js

```diff
@@ -81,7 +81,12 @@ test.describe( 'Payment setup task', () => {
 			.click()
 			.catch( () => {} );
 
-		if ( await page.isVisible( 'text=Offline payment methods' ) ) {
+		await page.waitForLoadState( 'networkidle' );
+
+		if (
+			( await page.isVisible( 'text=Offline payment methods' ) ) ||
+			( await page.isVisible( 'text=Additional payment gateways' ) )
```

Comment:
Odd that this would be a difference between environments.  Maybe it's a recent update to WC Admin?

Reply:
This one threw me off as well. Its definitely something to look into. I opted to add an extra check instead of changing the previous one because I wasn't sure when either would be true.